lib/cli-engine.js

```diff
@@ -489,7 +489,10 @@ CLIEngine.getFormatter = function(format) {
         try {
             return require(formatterPath);
         } catch (ex) {
-            return null;
+            if (ex.code === "MODULE_NOT_FOUND" && ex.message.indexOf("'" + formatterPath + "'") !== -1) {
```

Comment:
I'm not sure this is the right behavior. In all cases, it seems like we want to say "There was a problem loading 'formatterPath`" at the front of the message and then rethrow the error. Similar to: https://github.com/eslint/eslint/blob/master/lib/config/config-file.js#L146